ra-optimisation/search-tree-rao/src/test/java/com/powsybl/openrao/searchtreerao/searchtree/algorithms/SearchTreeBloomerTest.java

```diff
@@ -569,4 +574,88 @@ void testDontFilterNullOperator() {
 
         assertEquals(1, filteredNaCombination.size()); // no combination filtered, because null operator should not count
     }
+
+    @Test
+    void removeIncompatibleCombinations() {
```

Comment:
replace test setup with an actual CRAC creation, it would be easier to read & maintain than mocks